addons/account/models/account_move_line.py

```diff
@@ -892,12 +893,12 @@ def _get_computed_taxes(self):
                 tax_ids = self.move_id.company_id.account_purchase_tax_id
         else:
             # Miscellaneous operation.
-            tax_ids = False if self.env.context.get('skip_computed_taxes') else self.account_id.tax_ids
+            tax_ids = self.env['account.tax'] if self.env.context.get('skip_computed_taxes') else self.account_id.tax_ids
 
         if self.company_id and tax_ids:
             tax_ids = tax_ids.filtered(lambda tax: tax.company_id == self.company_id)
 
-        if tax_ids and self.move_id.fiscal_position_id:
+        if self.move_id.fiscal_position_id:
             tax_ids = self.move_id.fiscal_position_id.map_tax(tax_ids)
```

Comment:
If there is not tax_ids why do you need to mapped it?

Reply:
```
>>> move_lines = env["account.move.line"].search([
...     ("move_id.fiscal_position_id", "!=", False),
...     ("tax_ids", "=", False),
...     ("display_type", "not in", ["line_section", "line_note", "payment_term"]),
... ], limit=1)
>>> move_lines._get_computed_taxes()
account.tax()
>>> move_lines.move_id.fiscal_position_id.map_tax(env['account.tax'])
account.tax(42,)
```

Reply:
basically I want that last line to be executed

Reply:
For get it, I got it

`line.move_id.fiscal_position_id.map_tax(self.env["account.tax"])
account.tax(42,)`